7_deploy_osh/roles/suse-osh-setup/tasks/main.yml

```diff
@@ -8,8 +8,19 @@
   tags:
```

Comment:
Please remove this part of the change: https://github.com/SUSE-Cloud/socok8s/pull/93